couchdb.spec

```
@@ -17,7 +17,11 @@ Source0:       http://apache.mirrors.ovh.net/ftp.apache.org/dist/couchdb/source/
 Source1:       %{name}.service
 Source2:       usr-bin-couchdb
 
+%if 0%{?rhel} && 0%{?rhel} < 9
```

I think this is enough:
```suggestion
%if 0%{?rhel} < 9
```
But anyway, current version is good to me too.

---

:+1: